_topics/en/bls-signatures.md

```diff
@@ -0,0 +1,81 @@
+---
+title: BLS signatures
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Scripts and Addresses
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+#primary_sources:
+#    - title: Test
+#    - title: Example
+#      link: https://example.com
+
+## Optional.  Each entry requires "title" and "url".  May also use "feature:
+## true" to bold entry and "date"
+optech_mentions:
+  - title: Library announced for BLS signatures
+    url: /en/newsletters/2018/08/07/#library-announced-for-bls-signatures
+
+  - title: "Presentation: compact multisignatures for smaller blockchains"
+    url: /en/newsletters/2018/10/09/#compact-multi-signatures-for-smaller-blockchains
+
+  - title: "Using Bitcoin-compatible BLS signatures for Discree Log Contracts (DLCs)"
+    url: /en/newsletters/2022/08/17/#using-bitcoin-compatible-bls-signatures-for-dlcs
+
+  - title: "Question about schnorr signatures versus BLS signatures"
+    url: /en/newsletters/2023/01/25/#bls-signatures-vs-schnorr
+
+## Optional.  Same format as "primary_sources" above
+see_also:
+  - title: Scriptless multisignatures
+    link: topic multisignature
+
+## Optional.  Force the display (true) or non-display (false) of stub
+## topic notice.  Default is to display if the page.content is below a
+## threshold word count
+#stub: false
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  Boneh-Lynn-Shacham signatures (**BLS signatures**) are digital
```

Comment:
```suggestion
  Boneh–Lynn–Shacham signatures (**BLS signatures**) are digital
```

Reply:
Leaving as-is.

Reply:
It's wrong though? Authors of a work are combined with an en dash, not hyphen (or at least, I've never seen it done otherwise).

Reply:
Updated, but this is not a rule I care about.